dm/worker/subtask.go

```diff
@@ -306,7 +309,7 @@ func (st *SubTask) closeUnits() {
 	}
 	for i := cui; i < len(st.units); i++ {
 		u := st.units[i]
-		log.Infof("[syncer] closing process unit %s", u.Type())
+		st.l.Info("closing unit process", zap.Stringer("unit", cu.Type()))
```

Comment:
unit `Type` returned from pb definition is not the same with our regulation, such as `Sync` vs `binlog replication`.
Shall we add an adapter to change pb unit name to our defined unit name

Reply:
how about change protobuf later?

Reply:
seems we can only change the enum variable name in protobuf to get a different `enum name` value. I haven't found a way to unified the unit name via changing protobuf.

Reply:
change protobuf and unfiy to use protobuf variable